about.html

```diff
@@ -0,0 +1,143 @@
+    <br>
+    <h1 class="event-name">Energy Conference<br>
+      Goiânia, Brazil</h1>
+    <p class="event-description mobile-none">
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus, minima saepe incidunt quis ex eum voluptatem rem
+      unde fuga repudiandae iure? Animi porro laboriosam odio dolore quisquam nam veritatis iste.<br><br> Lorem ipsum
+      dolor sit amet consectetur adipisicing elit. Non deleniti laboriosam quas, consequuntur sint hic adipisci
+      perferendis nisi corrupti odit dolores similique fugit minus libero optio quos rerum suscipit quae.<br><br>
+      Similique corporis in illo delectus! Commodi ducimus cum quam expedita, qui sit nisi itaque, asperiores officiis
+      facilis in maxime nesciunt voluptatibus. Quaerat debitis ea quidem? Minus, cupiditate? Soluta, eaque
+      dolorum.<br><br>
+      Ea quos labore officiis explicabo possimus? Suscipit beatae aliquid ipsam ab nisi perferendis aut magni! A quos
+      voluptate, cum eius, dicta quisquam sapiente dolor ea possimus, dignissimos quod delectus ipsa.<br><br>
+      Fugiat facere, dolorum cumque provident nisi dolore. Quo tempora suscipit accusantium perspiciatis quos aliquid
+      tenetur inventore ullam ut quae? Vel optio inventore dignissimos repellat unde magni beatae! Molestiae, optio ab?
+    </p>
+    <p class="event-description desktop-none">Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus, minima
+      saepe incidunt quis ex eum voluptatem rem unde fuga repudiandae iure? Animi porro laboriosam odio dolore quisquam
+      nam veritatis iste.<br><br> Lorem ipsum dolor sit amet consectetur adipisicing elit. Non deleniti laboriosam
+      quas, consequuntur sint hic adipisci perferendis nisi corrupti odit dolores similique fugit minus libero optio
+      quos rerum suscipit quae.<br><br></p>
+  </section>
+  <!-- Contact-->
+  <section>
+    <div class="contact-event">
+      <h3 class="contact-event_title">
+        For more information about the Energy event, please contact us:
+      </h3>
+      <a href="mailto:name@email.com" class="contact-event_mail">energy2024@events.com</a>
+    </div>
+  </section>
+  <section class="logo-info">
+    <h2>World Energy</h2>
+
+  </section>
+  <!-- Partners -->
+  <section class="partners">
+    <h2>Partners</h2>
+    <div class="partner-logos">
+      <img src="assets/images/logo1.png" alt="logo" />
+      <img src="assets/images/logo2.png" alt="logo2" />
+      <img src="assets/images/logo3.png" alt="logo3" />
+      <img src="assets/images/logo4.png" alt="logo4" />
+      <img src="assets/images/logo5.png" alt="logo5" />
+    </div>
+  </section>
+
+  <!-- Footer and Copyrights-->
+  <footer class="footer-cnt">
+    <div class="footer-logo">
+      <img src="assets/images/advanced-energy-vector-logo.png" alt="logo" width="300" />
+    </div>
+    <div class="rights">
+      © 2024 Creative Energy Event. <br />
+      Some Rights Reserved.
+    </div>
+  </footer>
```

Comment:
- [ ] On desktop view the footer has a dark background and on mobile view, it has a white background. Please apply this change on both pages.
- [ ] Please try and reduce the font size of the page's content.

Reply:
Changes made on the footer color, but can you be more specific on the font size? Should that also be in the footer?